src/routes/get-started/ReportBug.tsx

```diff
@@ -10,13 +10,34 @@ import {
   GoATextArea
 } from "@abgov/react-components";
 import { ComponentContent } from "@components/component-content/ComponentContent.tsx";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { Octokit } from "octokit";
 import { useNavigate } from "react-router-dom";
   
 export default function ReportBugPage() {
+  interface IPackage {
+    name: string;
+    location: string;
+  }
+
   let navigate = useNavigate();
   
+  const [packages] = useState<IPackage[]>(
+    [
+      {
+        name: "web",
+        location: "/@abgov/web-components"
+      },
+      {
+        name: "angular",
+        location: "/@abgov/angular-components"
+      },
+      {
+        name: "react",
+        location: "/@abgov/react-components"
+      }
+    ],
+  )
```

Comment:
This doesn't need to be in a `useState`

Reply:
I had originally written this with a `setPackages` but then it was never used, so I just removed it, not thinking about changing the whole declaration. This is done.